package.json

```diff
@@ -102,14 +102,17 @@
   },
   "dependencies": {
     "@mui/icons-material": "^5.8.4",
+    "@mui/lab": "^5.0.0-alpha.88",
     "@mui/material": "^5.8.6",
     "electron-debug": "^3.2.0",
     "electron-log": "^4.4.7",
     "electron-updater": "^5.0.3",
     "lodash": "^4.17.21",
     "react": "^18.1.0",
     "react-dom": "^18.1.0",
-    "react-router-dom": "^6.3.0"
+    "react-markdown": "^8.0.3",
+    "react-router-dom": "^6.3.0",
+    "remark-gfm": "^3.0.1"
```

Comment:
`react-markdown` and `remark-gfm` are used to render Markdown with React. `remark-gfm` (recommended by `react-markdown`) enhances react-markdown by rendering more "advanced" markdown features such as checkboxes, tables, etc.